src/components/NavigationBar/NavigationBar.module.css

```diff
@@ -0,0 +1,13 @@
+.navbar {
```

Comment:
Stop committing css. You  should only be committing `.scss` files. You need to turn off this compilation process your turned on because it is useless and creating unwanted files.